documentation: Update form documentation with brand copy (#87)
ithaka/pharos: packages/pharos-site/static/guidelines/text-input.docs.mdx

  <PageSection subSectionLevel={1} title="Labels">
    <ul>
      <li>

>         The input label needs to help tell the user what is expected of them, but they don't need to

Reviewer: Suggestion is for grammar, but I think this could also be further clarified to indicate the nuance between "tell"/"help tell" and "instruct":

```suggestion
        The input label needs to tell the user what is expected of them, but it doesn't need to
```

<PageSection title="Variants">
  <PageSection subSectionLevel={1} title="Text">
    <p>
      The text variant is a single-line text field for generic text, used when there isn't a more
      specific input type available for collecting a value.
    </p>
    <Canvas>
      <PharosTextInput type="text" placeholder="text"></PharosTextInput>
    </Canvas>
  </PageSection>
  <PageSection subSectionLevel={1} title="Password">
    <p>
      A password input is a single-line text field whose value is obscured, used when you need to
      protect the user's content from being read.
    </p>
    <Canvas>
      <PharosTextInput type="password" value="password"></PharosTextInput>
    </Canvas>
  </PageSection>
  <PageSection subSectionLevel={1} title="Email">
    <p>

>       The email variant provides validation to ensure proper formatting of an email address. Use the

Reviewer: I would avoid using "variant" here unless we mean a `variant` property or a set of polymorphic components—`type` feels a bit more appropriate because it's the name of the HTML attribute involved, which you might want to use in the `number` and `tel` sections below as well:

```suggestion
      The email type provides validation to ensure proper formatting of an email address. Use the
```
